cookbook/lib/examples/gradient_bubbles.dart

```diff
@@ -86,12 +86,13 @@ class MessageBubble extends StatelessWidget {
             borderRadius: const BorderRadius.all(Radius.circular(16.0)),
             child: BubbleBackground(
               colors: [
-                message.isMine
-                    ? const Color(0xFF6C7689)
-                    : const Color(0xFF19B7FF),
-                message.isMine
-                    ? const Color(0xFF3A364B)
-                    : const Color(0xFF491CCB),
+                if (message.isMine) ...[
+                  const Color(0xFF6C7689),
+                  const Color(0xFF3A364B),
+                ] else ...[
+                  const Color(0xFF19B7FF),
+                  const Color(0xFF491CCB),
+                ],
```

Comment:
Consider pulling the `const`s out:

```suggestion
                if (message.isMine) ...const [
                  Color(0xFF6C7689),
                  Color(0xFF3A364B),
                ] else ...const [
                  Color(0xFF19B7FF),
                  Color(0xFF491CCB),
                ],
```